src/plugins/analysis-platform/modules/Asset/AssetOverview.tsx

```diff
@@ -1,32 +1,29 @@
 import {
   AuthContext,
   hasDomainRole,
-  useSearch,
   Loading,
+  IUIPlugin,
+  useDocument,
 } from '@development-framework/dm-core'
 import React, { ReactNode, useContext } from 'react'
 import { Button, Divider } from '@equinor/eds-core-react'
 import { AssetTable } from './components'
 import { Link, useLocation } from 'react-router-dom'
 import { TAsset } from '../../Types'
-import { EBlueprints } from '../../Enums'
-import { DEFAULT_DATASOURCE_ID } from '../../const'
 
-export const AssetOverview = (): ReactNode => {
+type TAssets = {
+  assets: TAsset[]
+}
```

Comment:
this type looks a bit strange. I guess we could just use TAsset[] instead of TAssets.

if the suggestion above in app/data/AnalysisPlatformDS/instances/AnalysisPlatform.json is implemented, I think useDocument will return TAsset[]